src/main/java/carpet/script/api/WorldAccess.java

```diff
@@ -1495,7 +1497,7 @@ else throw new InternalExpressionException("Attacking entity needs to be a livin
 
             boolean success = WorldTools.createWorld(cc.s.getServer(), worldKey, seed);
             if (!success) return Value.FALSE;
-            CarpetServer.settingsManager.notifyPlayersCommandsChanged();
+            CommandHelper.notifyPlayersCommandsChanged(cc.s.getServer());
             return Value.TRUE;
```

Comment:
no need to keep it - custom_dimension is not going back, at least until adding dimensions mid flight is not internally supported.

Reply:
Change was made before it got commented, I think it's out of scope to remove it here.